references.bib

```diff
@@ -459,6 +514,7 @@ @article{Sopov1970
     journal = {Ukrainskiui Geometricheskiui Sbornik},
     year = {1970},
     pages = {139–156},
+    url={},
```

Comment:
Why is this url empty?

Reply:
This has been removed.

Reply:
But we need a URL for every BibTeX entry.

Reply:
I have added the URL